docs/Advanced-Usage/Chip-Communication.rst

```diff
@@ -0,0 +1,76 @@
+.. _chip-communication:
+
+Communicating with the Chip/DUT
+===============================
+
+What good is a chip if it can't communicate with the outside world? Chipyard designs communicate to the outside world in
+one of two ways:
+
+* using the Front-End Server (FESVR)
+* using Rocket Chip's JTAG/DTM interface.
+
+Debugging with the Front-End Server (FESVR)
```

Comment:
Can you crib some docs from the last 250 course to teach this stuff?

Reply:
Can you give me a link?

Reply:
https://riscv.org/wp-content/uploads/2015/01/riscv-software-stack-bootcamp-jan2015.pdf

Reply:
damn that's... classic